### PR TITLE
Fix docker entry-point.sh

### PR DESCRIPTION
**Pull request by de-husk**
When running the stargaze docker image, I was getting an error from `docker/entry-point.sh`:
```
$ docker run --rm --name stargaze \
    -p 1317:1317 \
    -p 26656:26656 \
    -p 26657:26657 \
    -p 9090:9090 \
    --mount type=volume,source=stargaze_data,target=/root \
    publicawesome/stargaze:7.0.0 /data/entry-point.sh testing stars1ey69r37gfxvxg62sh4r0ktpuc46pzjrmkkj479

...

Error: unknown flag: --name
Usage:
  starsd gentx [key_name] [amount] [flags]

...
```

And now it properly spins up the node.

### CODE REVIEW

File: docker/Dockerfile.test [hunk 1]
```diff
@@ -3,7 +3,7 @@ RUN apk add -U --no-cache ca-certificates
 
 WORKDIR /root
 COPY ./bin/starsd /usr/bin/starsd
-COPY ./docker/test-entry-point.sh ./entry-point.sh
```
Comment (de-husk) [inline docker/Dockerfile.test:6]: I removed `docker/test-entry-point.sh` because it had the same contents as `docker/entry-point.sh`.

**Comment by jhernandezb** [comment]
Thank you! we usually overwrite our entry points in CI
```
  - name: stargaze
    image: publicawesome/stargaze:6.0.0
    commands:
      - ./scripts/ci/upgrade/setup-preinstalled-stargaze.sh
  ```
  Thats probably why it wasn't working or at least didn't notice.

**Review by jhernandezb — APPROVED** [review]

File: docker/entry-point.sh [hunk 2]
```diff
@@ -14,12 +14,12 @@ if [ -z "$2" ]; then
 fi
 
 # Build genesis file incl account for passed address
-coins="100000000000ustarx,100000000000ucredits"
+coins="10000000000000000stake"
```
Comment (jhernandezb) [inline docker/entry-point.sh:17]:
```suggestion
coins="10000000000000000ustars"
```

File: docker/entry-point.sh [hunk 3]
```diff
@@ -14,12 +14,12 @@ if [ -z "$2" ]; then
...
 starsd init --chain-id $CHAINID $CHAINID
 starsd keys add validator --keyring-backend="test"
 starsd add-genesis-account validator $coins --keyring-backend="test"
 starsd add-genesis-account $GENACCT $coins --keyring-backend="test"
-starsd gentx --name validator --amount 1000000ustarx --keyring-backend="test"
+starsd gentx validator 10000000000stake --chain-id $CHAINID --keyring-backend="test"
```
Comment (jhernandezb) [inline docker/entry-point.sh:22]:
```suggestion
starsd gentx validator 10000000000ustars --chain-id $CHAINID --keyring-backend="test"
```

**Review by jhernandezb — CHANGES_REQUESTED** [review]
some of our contracts make specific use of `ustars`, so making that the default, although it should probably be an argument to the entry point.

**Comment by de-husk** [comment]
> some of our contracts make specific use of `ustars`, so making that the default, although it should probably be an argument to the entry point.

Agreed, I'll make an env var that you can set with a default.

I also want to be able to pass in multiple genesis stars addresses, for bootstrapping multiple testing accounts easily. So Ill do that in this commit as well.

**Comment by jhernandezb** [comment]
@de-husk sounds good let me know when this ready and I can merge it

**Comment by de-husk** [comment]
> @de-husk sounds good let me know when this ready and I can merge it

Awesome! It's ready now

**Review by jhernandezb — APPROVED** [review]

**Comment by jhernandezb** [comment]
@de-husk thank you! tagged v7.0.1 so there should be a public docker image soon as well. https://ci.publicawesome.com/public-awesome/stargaze/3123/1/6

**Comment by de-husk** [comment]
> @de-husk thank you! tagged v7.0.1 so there should be a public docker image soon as well. https://ci.publicawesome.com/public-awesome/stargaze/3123/1/6

Hmm, this appears to still be using the older code?

```
$ docker run --rm -i -t publicawesome/stargaze:7.0.1  cat /data/entry-point.sh
#!/bin/sh

CHAINID=$1
GENACCT=$2

if [ -z "$1" ]; then
  echo "Need to input chain id..."
  exit 1
fi

if [ -z "$2" ]; then
  echo "Need to input genesis account address..."
  exit 1
fi

...
```

**Comment by jhernandezb** [comment]
Yeah my bad, tagged the wrong commit pushing new version now https://ci.publicawesome.com/public-awesome/stargaze/3124

**Comment by de-husk** [comment]
No worries, Thanks!

**Pull request merged**